src/routes/Routes.tsx

```diff
@@ -217,10 +217,11 @@ const Routes = () => {
           exact
           path={MEMBERS_ROUTE}
           component={ColonyMembers}
-          layout={NavBar}
+          layout={Complex}
           routeProps={({ colonyName }) => ({
             backText: ColonyBackText,
             backRoute: `/colony/${colonyName}`,
+            hasSubscribedColonies: true,
```

Comment:
Didn't we decide that ColonyMembers shouldn't have subscribedColonies list?

Reply:
Just because it looked weird, but now it looks good :)

Reply:
But yeah, you're right. I think I got carried away. I'll remove it.